app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceContractExtensions.kt

```diff
@@ -12,19 +14,24 @@ import com.hedvig.app.util.extensions.view.show
 import e
 import java.time.format.DateTimeFormatter
 
-fun InsuranceQuery.Contract.bindTo(binding: InsuranceContractCardBinding, marketManager: MarketManager) =
+fun InsuranceQuery.Contract.bindTo(
+    binding: InsuranceContractCardBinding,
+    marketManager: MarketManager
+) =
     binding.apply {
         status.fragments.contractStatusFragment.let { contractStatus ->
             contractStatus.asPendingStatus?.let {
                 firstStatusPill.show()
                 firstStatusPill.setText(R.string.DASHBOARD_INSURANCE_STATUS_INACTIVE_NO_STARTDATE)
+                typeOfContract.bindBackgroundColor(this)
             }
             contractStatus.asActiveInFutureStatus?.let { activeInFuture ->
                 firstStatusPill.show()
                 firstStatusPill.text = firstStatusPill.resources.getString(
                     R.string.DASHBOARD_INSURANCE_STATUS_INACTIVE_STARTDATE,
                     dateTimeFormatter.format(activeInFuture.futureInception)
                 )
+                removeBackgroundColor()
```

Comment:
While we're touching this code - I think we should use the gray background color that's been requested in other tickets instead of removing the background color 😄 It's linked in APP-667.